### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by zakrzaq**
- Docker
- docker-compose
- local, prod env

### CODE REVIEW

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  databrary-db:
+    image: postgres:16.2-alpine
```
[the remark this answers is not in the document]
Comment (zakrzaq) [reply docker-compose.prod.yml:5]: Will remove. This was more of a place holder more ten anything else. I would not dream of running prod db in docker

File: databrary/settings/alone.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+from .base import *  # NOQA
+
+DEBUG = True
+ALLOWED_HOSTS = ["*"]
+
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
```
[the remark this answers is not in the document]
Comment (zakrzaq) [reply databrary/settings/alone.py:13]: I left option to run BE repo standalone. I will replace with local instance of postgres for consistency

File: databrary/settings/base.py [hunk 3]
```diff
@@ -76,8 +77,12 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": ("django.db.backends.postgresql"),
```
[the remark this answers is not in the document]
Comment (zakrzaq) [reply databrary/settings/base.py:80]: We don't

File: databrary/settings/base.py [hunk 4]
```diff
@@ -77,7 +77,7 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": ("django.db.backends.postgresql"),
+        "ENGINE": "django.db.backends.postgresql",
```
Comment (OlhaHn) [inline databrary/settings/base.py:80]: What about the rest? "Name", "User" etc

**Review by OlhaHn — CHANGES_REQUESTED** [review]

File: databrary/settings/base.py (hunk 4 above)
Comment (zakrzaq) [reply databrary/settings/base.py:80]: removed

**Review by OlhaHn — APPROVED** [review]

**Pull request merged**